src/validations/user.validation.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from 'zod'
+
+export const roleValues = ['admin', 'user', 'moderator', 'guest'] as const
```

Comment:
Can we have a the role schema in one single place we are kind of duplicating the value a bit

we can have one schema for role as
const role =  z.enum([whatever_roles])
type Role = z.infer<typeof role>;

Instead of have multiple source of truth

Reply:
Done